agent/util-scripts/pbench-register-tool

```diff
@@ -250,6 +250,12 @@ if [[ ${remotes_arg::1} == "@" ]]; then
 		exit 1
 	fi
 
+	if [[ ! -s ${remotes_file} ]]; then
+		error_log "--remotes=@${remotes_file} specifies an empty file"
+		usage >&2
```

Comment:
Seems like issuing a usage message here will cloud the error.

Reply:
I followed the current "standard practice": both `pbench-register-tool` and `pbench-register-tool-set` print out their usage message on any error (although when `pbench-register-tool` is used in checking mode by `pbench-register-tool-set`, its usage message is suppressed: only the `pbench-register-tool-set`  usage is printed out).

Opened a discussion about this: #2552